pkg/vm/vm.go

```diff
@@ -208,7 +208,12 @@ func (v *VM) PrintOps() {
 			case opcode.INITSLOT:
 				desc = fmt.Sprintf("%d local, %d arg", parameter[0], parameter[1])
 			case opcode.SYSCALL:
-				desc = fmt.Sprintf("%q", parameter)
+				var name string
+				fp := v.GetInteropByID(binary.LittleEndian.Uint32(parameter))
```

Comment:
Perhaps we should use `GetInteropID(...)` function instead of `binary.LittleEndian.Uint32(parameter)`?

Reply:
Fixed.